EditImprovements/README.md

```diff
@@ -0,0 +1,16 @@
+EditImprovements - Edit.Locate and ~.Search display the target location in the first line of the viewer
+
+Description
+-----------
+
+When invoking them on small viewers, Edit.Locate and ~.Search sometimes position the text in the viewer that the target location is beyond the end of the viewer.
+This patch insures that the target location will always be at the first line of the viewer.
+
+Installation
+------------
+
+- Apply [`Edit.1r.patch`](Edit1r.patch) to `Edit.Mod` and [`EditU.0.patch`](Editu.0.patch) to `EditU.Mod`.
```

Comment:
```suggestion
- Apply [`Edit.1r.patch`](Edit.1r.patch) to `Edit.Mod`.

- In case you want to use the [UTF8CharsetLite][../UTF8CharsetLite/README.md) modification, apply [`EditU.0.patch`](EditU.0.patch) to `EditU.Mod`.
```

Both links are broken in the rendered markdown (you can verify if you click "View File" in the pull request view, or here), because links are case sensitive: https://github.com/schierlm/Oberon2013Modifications/blob/77f17013d6f422c80f26e8d8497a07ae594ad258/EditImprovements/README.md

Also, I would separate the EditU patch into a separate paragraph, as it depends on UTF8CharsetLite modification.